src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategy.kt

```diff
@@ -1190,7 +1190,7 @@ abstract class ManagedStrategy(
 
     private fun initializeVarHandleMethodCallTracePoint(
         tracePoint: MethodCallTracePoint,
-        varHandle: VarHandle,
+        varHandle: Any?, // for Java 8 VarHandle class does not exist
```

Comment:
Is it possible to make it `Any` (without `?`)? The same question applies to other `VarHandle -> Any?` type changes.